pkg/services/pluginsintegration/clientmiddleware/httpclient_middleware_test.go

```diff
@@ -262,6 +262,36 @@ func TestHTTPClientMiddleware(t *testing.T) {
 				require.Len(t, middlewares, 1)
 				require.Equal(t, forwardPluginRequestHTTPHeaders, middlewares[0].(httpclient.MiddlewareName).MiddlewareName())
 
+				reqClone := req.Clone(req.Context())
+				// Create a header on the request as if it had been set by some other logic e.g. preceding middleware
+				reqClone.Header.Set(backend.OAuthIdentityTokenHeaderName, "bearer test-token")
```

Comment:
it makes sense to do this in the other test "Should not overwrite an existing header", no?

Reply:
Oops I put this in the wrong test! Will fix 😊 nice catch!